src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -379,7 +379,7 @@ public SectionDetailsBundle getSectionForCourse(String section, String courseId)
         for (int i = 0; i < students.size(); i++) {
             StudentAttributes s = students.get(i);
             
-            if (section == null) {   // First student of first section
+            if (section == null) { // First student of first section
```

Comment:
Errr actually I'm causing inconsistencies with other part of this class (you can check it yourself) if I do it, so I'm opting out of this.